visier/api/model.py

```diff
@@ -0,0 +1,73 @@
+# This file is part of visier-connector-python.
+#
+# visier-connector-python is free software: you can redistribute it and/or modify
+# it under the terms of the Apache License, Version 2.0 (the "License").
+#
+# visier-connector-python is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# Apache License, Version 2.0 for more details.
+#
+# You should have received a copy of the Apache License, Version 2.0
+# along with visier-connector-python. If not, see <https://www.apache.org/licenses/LICENSE-2.0>.
+
+"""
+API client for the Visier Model API.
+"""
+
+from typing import List
+import urllib.parse as urlparse
+from requests import Response
+from visier.connector import SessionContext
+from .base import ApiClientBase
+
+
+class ModelApiClient(ApiClientBase):
+    """API client for the Visier Model API."""
+    def get_analytic_objects(self, ids: List[str] = None) -> Response:
+        """Get analytic objects by id"""
+        def call_impl(context: SessionContext) -> Response:
+            args = _ids_as_url_args(ids)
+            url = context.mk_url(f"/v1/data/model/analytic-objects{args}")
+            return context.session().get(url)
+        return self.run(call_impl)
+
+    def get_metrics(self, ids: List[str] = None) -> Response:
+        """Get metrics by id"""
+        def call_impl(context: SessionContext) -> Response:
+            args = _ids_as_url_args(ids)
+            url = context.mk_url(f"/v1/data/model/metrics{args}")
+            return context.session().get(url)
+        return self.run(call_impl)
+
+    def get_dimensions(self, object_id: str, ids: List[str] = None) -> Response:
```

Comment:
Should we include the metric, properties, selection concept variations of this function?

Reply:
Not properties, but yes, I'll add the dimensions and selection-concepts as keyed by a metric